.vsts-ci.yml

```diff
@@ -50,8 +50,8 @@ phases:
         Build_Release:
           _BuildConfig: Release
           _PublishType: none
-          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+          ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
             _SignType: test
-          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI') }}:
```

Comment:
Is this supposed to be

```
${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
```

For that matter, you can change the `in` in the condition above to an `eq` to do a string comparison rather than a hash key check.